ethers-core/src/utils/geth.rs

```diff
@@ -364,7 +372,8 @@ impl Geth {
         // Open the HTTP API
         cmd.arg("--http");
         cmd.arg("--http.port").arg(port.to_string());
-        cmd.arg("--http.api").arg(API);
+        let apis = if self.enable_debug { format!("{},debug", API) } else { API.to_owned() };
+        cmd.arg("--http.api").arg(apis);
```

Comment:
I would like there to be a `Vec<String>` for all additional modules. so we can also enable admin for example.

Should this also be enabled for `ws.api`?

Reply:
Yes. You are right.

Reply:
> I would like there to be a `Vec<String>` for all additional modules. so we can also enable admin for example.

Do you want me to add another method like `enable_admin`?

Reply:
@mattsse I made some changes. Take a look when you are available. Thanks.